addons/point_of_sale/tools/posbox/overwrite_before_init/etc/init_posbox_image.sh

```diff
@@ -139,7 +147,10 @@ PIP_TO_INSTALL="
     Werkzeug==2.0.2 \
     urllib3==1.26.5 \
     pyOpenssl==22.0.0 \
-    cryptography==36.0.2"
+    cryptography==36.0.2 \
+    screeninfo \
+    zeep \
+    num2words"
 
 pip3 install ${PIP_TO_INSTALL} --break-system-package
```

Comment:
wouldn't it be a good occasion to also introduce a "iot-requirement.txt" file 

It could also be more practical to do one common requirement file for windows iot and iot-box as currently the windows one have its dedicated requirement file:
https://github.com/odoo/odoo/blob/17.0/setup/win32/requirements-local-proxy.txt
(note that you can have condition in requirement.txt to install some lib only on windows, see: https://stackoverflow.com/a/35614580 )